src/Controllers/AddTaskController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace App\Controllers;
+use App\Models\TasksModel;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Psr\Http\Message\ResponseInterface as Response;
+
+/**
+ * Class AddTaskController
+ * @package App\Controllers
+ * @return Response with header
+ */
+class AddTaskController {
+    protected TasksModel $model;
+    /**
+     * AddTaskController constructor.
+     * @param TasksModel $model
+     */
+    public function __construct (TasksModel $model)
+    {
+        $this->model = $model;
+    }
+    /**
+     * @param Request $request
+     * @param Response $response
+     * @param $args
+     * @return Response
+     */
+    public function __invoke(Request $request, Response $response, $args): Response
+    {
+        $data = $request->getParsedBody();
+        $newTask = $data['task_name'];
+        if ($newTask != "") {
```

Comment:
I would suggest checking `isset($data['task_name'])` here instead, just in case someone somehow ends up on this route and they haven't submitted the form. In it's current state this would cause an error on line 31 (undefined index 'task_name')